src/main/java/com/s2u2m/kit/common/enumtype/layer/EnumLayerable.java

```diff
@@ -0,0 +1,61 @@
+/*
+ *    Copyright [2018] [s2u2m]
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.s2u2m.kit.common.enumtype.layer;
+
+import java.text.MessageFormat;
+
+/**
+ * Make enum in multiple layers to make definition clear.
+ * @author Amos Xia
+ */
+interface EnumLayerable extends EnumLayerDefine {
```

Comment:
Error, cannot use EnumLayerDefine for a part of nodes in same layer, must define for whole layer.
or it will meet same serialize value for different nodes.